ops/setup-environment.sh

```diff
@@ -14,15 +14,19 @@ readonly fortis_interface_host="${8}"
 readonly eh_conn_str="${9}"
 readonly feature_service_db_conn_str="${10}"
 readonly fortis_central_directory="${11}"
+readonly sb_conn_str="${12}"
 
 readonly fortis_admin_interface="http://${fortis_interface_host}/#/site/${site_name}/admin"
 readonly default_language="en"
 readonly checkpoint_directory="HDFS://"
 readonly eh_path="published-messages"
 readonly eh_consumer_group="\$Default"
+readonly sb_queue_site="siteMutations"
+readonly sb_queue_streams="streamMutations"
 readonly fortis_models_directory="${fortis_central_directory}/sentiment/"
 
-kubectl create configmap "${spark_config_map_name}" --namespace spark --from-literal=FORTIS_CASSANDRA_HOST="${cassandra_host}" --from-literal=FORTIS_FEATURE_SERVICE_HOST="${feature_service_host}" --from-literal=DEFAULT_SITE_NAME="${site_name}" --from-literal=FORTIS_APPINSIGHTS_IKEY="${app_insights_id}" --from-literal=SPARK_DAEMON_MEMORY="${SPARK_DAEMON_MEMORY}" --from-literal=HA_PROGRESS_DIR="${checkpoint_directory}" --from-literal=DEFAULT_LANGUAGE="${default_language}" --from-literal=FORTIS_SERVICE_HOST="${graphql_service_host}" --from-literal=FORTIS_MODELS_DIRECTORY="${fortis_models_directory}" --from-literal=PUBLISH_EVENTS_EVENTHUB_CONNECTION_STRING="${eh_conn_str}" --from-literal=PUBLISH_EVENTS_EVENTHUB_PATH="${eh_path}" --from-literal=PUBLISH_EVENTS_EVENTHUB_PARTITION="${eh_consumer_group}"
+kubectl create configmap "${spark_config_map_name}" --namespace spark --from-literal=FORTIS_CASSANDRA_HOST="${cassandra_host}" --from-literal=FORTIS_FEATURE_SERVICE_HOST="${feature_service_host}" --from-literal=DEFAULT_SITE_NAME="${site_name}" --from-literal=FORTIS_APPINSIGHTS_IKEY="${app_insights_id}" --from-literal=SPARK_DAEMON_MEMORY="${SPARK_DAEMON_MEMORY}" --from-literal=HA_PROGRESS_DIR="${checkpoint_directory}" --from-literal=DEFAULT_LANGUAGE="${default_language}" --from-literal=FORTIS_SERVICE_HOST="${graphql_service_host}" --from-literal=FORTIS_MODELS_DIRECTORY="${fortis_models_directory}" --from-literal=SERVICE_BUS_CONNECTION_STRING="${sb_conn_str}" --from-literal=SERVICE_BUS_QUEUE_SITE="${sb_queue_site}" --from-literal=SERVICE_BUS_QUEUE_STREAM="${sb_queue_streams}"
```

Comment:
Nit: would be good to split this over multiple lines using `\` to make it clearer what the diffs are. You can copy paste the below:

```sh
kubectl create configmap "${spark_config_map_name}" --namespace spark \
  --from-literal=FORTIS_CASSANDRA_HOST="${cassandra_host}" \
  --from-literal=FORTIS_FEATURE_SERVICE_HOST="${feature_service_host}" \
  --from-literal=DEFAULT_SITE_NAME="${site_name}" \
  --from-literal=FORTIS_APPINSIGHTS_IKEY="${app_insights_id}" \
  --from-literal=SPARK_DAEMON_MEMORY="${SPARK_DAEMON_MEMORY}" \
  --from-literal=HA_PROGRESS_DIR="${checkpoint_directory}" \
  --from-literal=DEFAULT_LANGUAGE="${default_language}" \
  --from-literal=FORTIS_SERVICE_HOST="${graphql_service_host}" \
  --from-literal=FORTIS_MODELS_DIRECTORY="${fortis_models_directory}" \
  --from-literal=SERVICE_BUS_CONNECTION_STRING="${sb_conn_str}" \
  --from-literal=SERVICE_BUS_QUEUE_SITE="${sb_queue_site}" \
  --from-literal=SERVICE_BUS_QUEUE_STREAM="${sb_queue_streams}" \
  --from-literal=PUBLISH_EVENTS_EVENTHUB_CONNECTION_STRING="${eh_conn_str}" \
  --from-literal=PUBLISH_EVENTS_EVENTHUB_PATH="${eh_path}" \
  --from-literal=PUBLISH_EVENTS_EVENTHUB_PARTITION="${eh_consumer_group}"
```

Also: looks like line 29 should be a continuation of line 28 -- the snippet above fixes that.